onnxruntime/python/tools/transformers/models/llama/benchmark.py

```diff
@@ -638,9 +626,9 @@ def get_args():
     if "ort" in args.benchmark_type:
         setattr(args, "execution_provider", f"{args.device.upper()}ExecutionProvider")  # noqa: B010
         if args.execution_provider == "CUDAExecutionProvider":
-            args.execution_provider = (args.execution_provider, {"device_id": args.device_id})
+            args.execution_provider = (args.execution_provider, {"device_id": rank})
```

Comment:
This may be a problem when wanting to benchmark on different GPU device ids. If `rank = 0` for the 7B and 13B models, then the benchmarking will only happen with `GPU device id = 0`. We need to keep the option of benchmarking those models with different device ids (e.g. one benchmark with device id = 3, another benchmark with device id = 7). We should also be able to benchmark the 70B model with a different set of GPU device ids (e.g. 2, 3, 4, 5 vs 0, 1, 2, 3).

Reply:
Yes, I think CUDA_VISIBLE_DEVICES will do that.
For single gpu we can run benchmark like CUDA_VISIBLE_DEVICES=0(or 1,2,3.. any gpu you like) python -m ...
For multiple gpus we can do CUDA_VISIBLE_DEVICES=0,1,2,5 bash run 4 ....